### PR TITLE
IGAPP-1253: Upgrade Node to 18 LTS

### PR DESCRIPTION
**Pull request by sarahsporck**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: tools/package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "ts:check": "tsc --build"
   },
   "devDependencies": {
-    "@octokit/auth-app": "^4.0.4",
+    "@octokit/auth-app": "^4.0.9",
```
Comment (sarahsporck) [inline tools/package.json:16]: yarn audit discovered a vulnerability in one of this packages subdepenedncies

File: web/package.json [hunk 2]
```diff
@@ -2,10 +2,6 @@
   "name": "web",
   "version": "0.1.0",
   "private": true,
-  "engines": {
-    "node": ">=10.13.0",
-    "npm": ">=3.8"
-  },
```
Comment (sarahsporck) [inline web/package.json:8]: we never really track/care about these values

File: web/package.json [hunk 3]
```diff
@@ -111,7 +107,7 @@
     "jest-fetch-mock": "^3.0.3",
     "jest-junit": "^15.0.0",
     "moment-locales-webpack-plugin": "^1.2.0",
-    "moment-timezone-data-webpack-plugin": "1.5.0",
+    "moment-timezone-data-webpack-plugin": "1.5.1",
```
Comment (sarahsporck) [inline web/package.json:110]: required for node update

**Review by LeandraH — APPROVED** [review]
Looks good, tested in Firefox, real iPhone and simulated Android.

**Pull request merged**